src/lib/y2storage/proposal/space_distribution.rb

```diff
@@ -47,7 +49,9 @@ def initialize(volumes_by_disk_space)
         end
         @spaces.freeze
         spaces_by_disk.each do |disk, spaces|
-          set_num_logical_for(spaces, disk.partition_table)
+          disk.as_not_empty do
```

Comment:
What happens if the disk is completely empty? AFAICS that means that its whole free space is disregarded, right?